src/Recoil.rei

```diff
@@ -84,6 +101,16 @@ type value('value) = 'value;
 external useRecoilValue: t('value, 'mode) => value('value) =
   "useRecoilValue";
 
+type loadableValue('a) = {
+  state: State.t,
+  getValue: (. unit) => 'a,
+  toPromise: (. unit) => Js.Promise.t('a),
+};
+
+[@bs.module "recoil"]
+external useRecoilValueLoadable: readOnly('value) => loadableValue('value) =
```

Comment:
I _think_ that it accepts a `t('value, 'mode)` and not only a `readOnly('value)`